spack/packages/shumlib/package.py

```diff
@@ -0,0 +1,92 @@
+# Copyright 2013-2022 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+import os
+import shutil
+from spack.package import MakefilePackage
+
+
+class Shumlib(MakefilePackage):
+
+    """Shared UM Library.
+
+    Shumlib is the collective name for a set of libraries which are
+    used by the UM; the UK Met Office's Unified Model, that may be of
+    use to external tools or applications where identical
+    functionality is desired. The hope of the project is to enable
+    developers to quickly and easily access parts of the UM code that
+    are commonly duplicated elsewhere, at the same time benefiting
+    from any improvements or optimisations that might be made in
+    support of the UM itself.
+    """
+
+    homepage = "https://code.metoffice.gov.uk"
+    url = "https://github.com/metomi/shumlib/archive/refs/tags/um13.0.tar.gz"
+
+    # shumlib does not currently build in parallel
+    parallel = False
+
+    version(
+        "13.0",
+        sha256="50f43a2f8980e8fbeafd053376612503bcb17c34948297f19b2c95ce0642b340",
+    )
+
+    variant("openmp", default=False, description="enable OpenMP support")
+
+    def edit(self, spec, prefix):
+
+        """Minor setup edits."""
+
+        # Create a copy the makefile
+        source = "make/vm-x86-gfortran-gcc.mk"
```

Comment:
Do we need to change this to the `make/nci-x86-ifort-icc.mk` version, to be consistent with the `ifort` compiler being used in the `intel-openmpi` container? Or is this un-ncessary if we are getting the compiler via the `FC` compiler flag